### ISSUE TITLE
Support arbitrary blocks

### ISSUE THREAD

**Issue by TheMadPrompter** [amake/orgro#79]
In Org mode, [everything between #+begin... and #+end... is considered a block.](https://orgmode.org/manual/Blocks.html) While the source code blocks are the most common, arbitrary blocks can be created. They are often used to create self-contained collapsible sections independent of the header hierarchy.

In my experience, the only 'common' blocks like #+begin_src or #+begin_quote are treated by the app as blocks and can be folded. I use a lot of blocks with other names, so not being able to fold them in the app like I can in Emacs is pretty inconvenient.

**Comment by amake** [comment]
I didn't support this because it's actually a minor pain to parse, but I can do it.

Just to be extra sure, can you provide a snippet with a block that you would like supported?

**Comment by TheMadPrompter** [comment]
> Just to be extra sure, can you provide a snippet with a block that you would like supported?

Some of the common blocks I use look like

```
#+BEGIN_RESULTS 
Text
#+END_RESULTS
```
and
```
#+BEGIN_DESCRIPTION
Text
#+END_DESCRIPTION
```

**Comment by amake** [comment]
This will be available in v1.31.0. Look for it soon:

- [iOS](https://testflight.apple.com/join/8YTxcuZC)
- [Android](https://play.google.com/store/apps/details?id=com.madlonkay.orgro)

**Comment by amake** [comment]
This is now available in v1.31.2.
